common/app/common/PageContentBean.java

```diff
@@ -0,0 +1,19 @@
+package common;
+
+import common.controllers.PageContent;
+
+public abstract class PageContentBean extends PageContent {
+    private String additionalTitle;
+
+    public PageContentBean() {
+    }
+
+    @Override
+    public final String getAdditionalTitle() {
+        return additionalTitle;
+    }
+
+    public void setAdditionalTitle(final String additionalTitle) {
+        this.additionalTitle = additionalTitle;
+    }
```

Comment:
You can implement directly these methods in PageContent, there's no need for one more class.